src/store.rs

```diff
@@ -434,6 +444,12 @@ impl<E: Element> Store<E> for DiskStore<E> {
         Ok(true)
     }
 
+    fn delete(config: StoreConfig) -> Result<bool> {
+        remove_file(StoreConfig::data_path(&config.path, &config.id))?;
+
+        Ok(true)
```

Comment:
what does the `bool` indicate, it seems to always return `true`

Reply:
Yes, it does.  I'll update it to pass through the result of the remove_file call then `Result<()>`.